controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -470,6 +473,14 @@ def test_create_volume_with_get_volume_illegal_object_name_exception(self, stora
         self.assertTrue(msg in self.context.details)
         self.mediator.get_volume.assert_called_once_with(volume_name, pool_id='pool1')
 
+    @patch("controller.controller_server.csi_controller_server.get_agent")
+    def test_create_volume_with_prefix_too_long_exception(self, storage_agent):
+        storage_agent.return_value = self.storage_agent
+        self.request.parameters.update({"volume_name_prefix": "a" * 128})
+        self.servicer.CreateVolume(self.request, self.context)
+
+        self.assertEqual(self.context.code, grpc.StatusCode.INVALID_ARGUMENT)

```

Comment:
is this related to the PR?

Reply:
it was a gap that I saw and it was quick to fix